data-common.json

```diff
@@ -49,37 +49,37 @@
     "sharedmem": {
       "val": "flagged",
       "note_id": "firefox-sharedmem",
-      "note_html": "The feature have to be enabled via \"javascript.options.shared_memory\" setting under <code>about:config</code>.  It is enabled by default in Firefox Developer Edition and Firefox Nightly builds."
+      "note_html": "The feature have to be enabled via <code>javascript.options.shared_memory</code> setting under <code>about:config</code>.  It is enabled by default in Firefox Developer Edition and Firefox Nightly builds."
     },
     "nomob": {
-      "val": "flagged",
+      "val": true,
       "note_id": "firefox-nomob",
       "note_html": "The feature is available on desktop versions only, it is not available on mobile versions yet."
     },
     "developer": {
-      "val": "flagged",
+      "val": false,
       "note_id": "firefox-developer",
       "note_html": "The feature is available only in Firefox Developer Edition and Firefox Nightly builds."
     },
     "nightly": {
-      "val": "flagged",
+      "val": false,
```

Comment:
I'm not sure that it's a good idea.

Reply:
It depends on what "flagged" means.  To me it means that the feature is not available by default, but it can be turned on by the user (either by changing the browser configuration or adding a command line parameter).  This is not what nightly means: it is available by default on nightly builds (via compilation flags), and not available at all on other builds.  Changing this to "flagged" marked lots of features as available on old builds, which is not true.